docs/reference/openapi.yaml

```diff
@@ -252,7 +252,7 @@ components:
       type: object
 info:
   title: BlueAPI Control
-  version: 0.0.5
+  version: 0.0.6
```

Comment:
Should: Given the addition of a new required parameter I think this is 0.1.0, but since I'm arguing that it should be optional in another comment, leave until that's resolved.